code/game/objects/items/cigs_lighters.dm

```diff
@@ -349,7 +349,7 @@ CIGARETTE PACKETS ARE IN FANCY.DM
 	type_butt = /obj/item/cigbutt/roach
 	throw_speed = 0.5
 	item_state = "spliffoff"
-	smoketime = 120 // four minutes
+	smoketime = 4 * 60
```

Comment:
use the defines

Reply:
its because its not seconds, its using deciseconds time the delta_time